doc/source/whatsnew/v1.0.0.rst

```diff
@@ -816,6 +816,7 @@ I/O
 - Bug in :func:`read_json` where default encoding was not set to ``utf-8`` (:issue:`29565`)
 - Bug in :class:`PythonParser` where str and bytes were being mixed when dealing with the decimal field (:issue:`29650`)
 - :meth:`read_gbq` now accepts ``progress_bar_type`` to display progress bar while the data downloads. (:issue:`29857`)
+- :func:`read_excel` now accepts binary data (:issue:`15914`)
```

Comment:
can anyone clarify the most appropriate between `:func:` and `:meth:`?

Reply:
`:func:` is fine.